include/powsybl/iidm/Extendable.hpp

```diff
@@ -53,6 +54,16 @@ class Extendable {
 
     stdcxx::range<Extension> getExtensions();
 
+    /**
+     * Returns an extensionAdder to build and add an extension for this extendable.
```

Comment:
Fix the name of the `extensionAdder` to make a pretty link in doxygen. Same with extension and extendable.